src/main/kotlin/exercise2/task1/FindPairOfHighestSum.kt

```diff
@@ -18,7 +18,60 @@ import org.jetbrains.exercise2.task3.findPairWithBiggestDifference
  */
 
 internal fun List<Int>.findHighestSumPair(): Pair<Int, Int> {
-    TODO("Implement me!!")
+
+
+//    for ( i in 0  ..this.size - 1){
```

Comment:
Please clean up all the comments before pushing the changes and creating the PR.